data/korath/korath ships.txt

```diff
@@ -15,62 +16,69 @@ ship "Korath Dredger"
 	attributes
 		category "Heavy Freighter"
 		"cost" 29370000
-		"shields" 39000
-		"hull" 16000
-		"required crew" 201
-		"bunks" 395
+		"shields" 32500
+		"hull" 16500
+		"required crew" 137
+		"bunks" 357
+		"crew" 315
```

Comment:
I will go check again to see if it works, but I do have ships in missions that use this to force full bunks without having to set that variant ship to have several times more crew than normal.

Reply:
My initial testing with a pair of modified albatrosses shows it not having an effect, which is odd since I recall it clearly having an effect with the assault dredgers. I'll have to do some more digging later to find out why they worked last time I tested those missions and isn't working here now.

Reply:
That attribute is currently at the wrong place, it should go one layer above like this:

```
ship X
    crew 123
    attributes
        ...
```